release/_releases_json.sh

```diff
@@ -1,8 +1,24 @@
 #!/bin/bash
 
-function regenerate_releases_json {
-	_process_product dxp
-	_process_product portal
+function generate_releases_json {
+	if [ "${1}" = "regenerate" ]
+	then
+		_process_product dxp
+		_process_product portal
+	else
+		_process_new_product dxp
+
+		local status_dxp=$?
+
+		_process_new_product portal
+
+		local status_portal=$?
+	fi
+
+	if [ "${status_dxp}" -eq "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ] && [ "${status_portal}" -eq "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
```

Comment:
@Wykenz, shouldn't we use `||` instead `&&`?

Reply:
Thank you for pointing this out, yes the way I imagined this solution it was supposed to be `&&` but we don't actually need all this for checking only one new version so I made some changes with using `"${LIFERAY_RELEASE_PRODUCT_NAME}"` and sendt a new [PR](https://github.com/natocesarrego/liferay-docker/pull/99)